### PR TITLE
Adds a step to configure the publish scope.

### LINKED ISSUE #7298: Missing step for setting publish scope in packages/quickstart.md
### What is the current behavior?

The package quickstart guide does not mention setting the publish scope.

### What changes are you suggesting?

Adding a step to show how to configure the publish scope will improve the success rate for users following this guide.

### PR DESCRIPTION
**Pull request by radfahrer**
### Why:

There is a missing step.
Closes #7298 

### What's being changed:

Adds missing step.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@radfahrer Thanks so much for opening a PR! I'll get this triaged for review 💖

File: content/packages/quickstart.md [hunk 1]
```diff
@@ -86,13 +86,25 @@ In this guide, you'll create a {% data variables.product.prodname_actions %} wor
...
+   - or using the `publishConfig` key in `package.json`
+      ```json
+      "publishConfig": {
+        "@YOUR-USERNAME:registry": "https://npm.pkg.github.com"
+      }
+      ```
+  
+9. Commit and push your changes to {% data variables.product.prodname_dotcom %}.
     ```shell
     $ git add .github/workflows/release-package.yml
```
Comment (felicitymay) [inline content/packages/quickstart.md:103]: We need to add either the new `.npmrc` file or the change to `packages.json` here too.

```suggestion
    $ git add .github/workflows/release-package.yml
    # Also add the file you created or edited in the previous step.
    $ git add <em>.npmrc or package.json</em>
```

**Review by felicitymay — COMMENTED** [review]
Thank you for spotting this omission from the quickstart guide and adding instructions for how to fix it. 💖 

I'm a packages novice and tried the quick start guide with and without your step. I would have been baffled about how to fix this on my own. With your changes I was able to work it out 🙂

**Comment by felicitymay** [comment]
Apologies for the delay in getting to your review, it's been a busy time. I'm going to check that my text changes look okay on staging when it finishes deploying: https://docs-7299--patch-2.herokuapp.com/en/packages/quickstart. As long as I haven't introduced any bugs, I'll update the branch from `main` and merge your changes.

Thank you so much for taking the time to raise this PR to fix this omission in the quickstart ✨

File: content/packages/quickstart.md [hunk 2]
```diff
@@ -86,13 +86,27 @@ In this guide, you'll create a {% data variables.product.prodname_actions %} wor
...
+   - Add an NPM configuration file for the repository by creating a `.npmrc, in the root directory with the contents:
+      ```
+      @<em>YOUR-USERNAME</em>:registry=https://npm.pkg.github.com
+      ```
+   - Edit the `package.json` file and specify the `publishConfig` key:
+      ```json
+      "publishConfig": {
+        "@<em>YOUR-USERNAME</em>:registry": "https://npm.pkg.github.com"
+      }
+      ```
```
Comment (felicitymay) [inline content/packages/quickstart.md:99]: 🤦🏻 
```suggestion
   - Add an NPM configuration file for the repository by creating a `.npmrc` file, in the root directory with the contents:
      {% raw %}
      ```
      @<em>YOUR-USERNAME</em>:registry=https://npm.pkg.github.com
      ```
      {% endraw %}
   - Edit the `package.json` file and specify the `publishConfig` key:
      {% raw %}
      ```json
      "publishConfig": {
        "@<em>YOUR-USERNAME</em>:registry": "https://npm.pkg.github.com"
      }
      ```
      {% endraw %}
```

**Review by felicitymay — APPROVED** [review]

**Pull request merged**